transactionoutbox-core/src/test/java/com/gruelbox/transactionoutbox/acceptance/TestMySql8.java

```diff
@@ -14,7 +14,7 @@
   @Container
   @SuppressWarnings("rawtypes")
   private static final JdbcDatabaseContainer container =
-      new MySQLContainer<>("mysql:8").withStartupTimeout(Duration.ofHours(1));
+      new MySQLContainer<>("mysql:8.0.27").withStartupTimeout(Duration.ofMinutes(5));
```

Comment:
The intent here is to break the test if it fails against the latest mysql.

What's the thinking here?

Reply:
I couldn't get test containers to start with 8. Seemed to work once with 8.0.27.